core/src/banking_stage/transaction_scheduler/scheduler_controller.rs

```diff
@@ -102,8 +107,9 @@ impl SchedulerController {
             self.timing_metrics.update(|timing_metrics| {
                 saturating_add_assign!(timing_metrics.decision_time_us, decision_time_us);
             });
-
             let new_leader_slot = decision.bank_start().map(|b| b.working_bank.slot());
+            self.leader_detection_metrics
+                .update_and_maybe_report(decision.bank_start());
             self.count_metrics
                 .maybe_report_and_reset_slot(new_leader_slot);
             self.timing_metrics
```

Comment:
nit: some nitpick ordering inconsistency.

type fields and initializer fields are ordered as `count_metrics` => `timing_metrics` => `leader_detection_metrics`.

however, this invocation code is ordered as `leader_detection_metrics` => `count_metrics` => `timing_metrics`.

Reply:
[a1fc272](https://github.com/anza-xyz/agave/pull/405/commits/a1fc272566a009c59b6a7f188c24f3fa385ffbb8)